app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -155,6 +159,17 @@ class EventDetailsFragment : Fragment() {
         startActivity(intent)
     }
 
+    private fun reportEvent(event: Event){
+        val email ="open.test24@gmail.com"
```

Comment:
@iamareebjamal this can be changed to some official fossasia email when the app is released in playstore
(for now using the common developer email that we are using for testing purpose)

Reply:
FOSSASIA already has official email for eventyay. Use that